DEV-9/SymbolReplacementInStrings/StringLine.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+
+namespace SymbolReplacementInStrings
+{
+    class StringLine
```

Comment:
Bad idea to join string and Random generator in one class and call it StringLine